backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/entities/Shop.java

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.google.gpay.virtualqueue.backendservice.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class Shop {
+    private String shopOwnerUid;
```

Comment:
Why not just call it shopId ?

Reply:
I'm not sure here, Ishita will be able to explain this better. What I understand is that, for one shop, we have two different Ids. One is shopId (Id of shop) and the other is shopOwnerUid (Unique Id of shop-owner).

Reply:
yeah resolved that!